From 4ad7cde759d6fec4924d12db1c72b411ea957e00 Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Tue, 4 Feb 2014 13:46:59 -0800
Subject: [PATCH 572/974] mmc: octeon_mmc: Disable debug printing.

... And fix octeon_mmc_dbg() definition to be correct C.

Signed-off-by: David Daney <david.daney@cavium.com>
[Original patch taken from Cavium SDK 3.1.2-568]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/mmc/host/octeon_mmc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mmc/host/octeon_mmc.c b/drivers/mmc/host/octeon_mmc.c
index 7fbfc06..fbcefa4 100644
--- a/drivers/mmc/host/octeon_mmc.c
+++ b/drivers/mmc/host/octeon_mmc.c
@@ -117,10 +117,10 @@ MODULE_PARM_DESC(ddr,
 		 "enable DoubleDataRate clocking:"
 		 " 0=no, 1=always, 2=at spi-max-frequency/2");
 
-#if 1
+#if 0
 #define octeon_mmc_dbg trace_printk
 #else
-static inline octeon_mmc_dbg(const char *s, ...) { }
+static inline void octeon_mmc_dbg(const char *s, ...) { }
 #endif
 
 static void octeon_mmc_acquire_bus(struct octeon_mmc_host *host)
-- 
2.6.2

